grafeas-client/CHANGELOG.md

```diff
@@ -1,4 +1,14 @@
 # Release History
+### 0.3.0 / 2019-08-20
```

Comment:
Because the only changes are to the non-inline documentation, I think this should be a tiny release. Thoughts?